src/api/rpc_api/state.rs

```diff
@@ -188,85 +184,85 @@ where
 
 	fn get_storage_keys_paged(
 		&self,
-		prefix: Option<StorageKey>,
+		storage_key: Option<StorageKey>,
 		count: u32,
 		start_key: Option<StorageKey>,
 		at_block: Option<Runtime::Hash>,
 	) -> Result<Vec<StorageKey>> {
 		let storage = self
 			.client()
-			.request("state_getKeysPaged", rpc_params![prefix, count, start_key, at_block])?;
+			.request("state_getKeysPaged", rpc_params![storage_key, count, start_key, at_block])?;
 		Ok(storage)
 	}
 
-	fn get_opaque_storage_by_key_hash(
+	fn get_opaque_storage_by_storage_key(
 		&self,
-		key: StorageKey,
+		storage_key: StorageKey,
 		at_block: Option<Runtime::Hash>,
 	) -> Result<Option<Vec<u8>>> {
 		let storage: Option<StorageData> =
-			self.client().request("state_getStorage", rpc_params![key, at_block])?;
+			self.client().request("state_getStorage", rpc_params![storage_key, at_block])?;
 		Ok(storage.map(|storage_data| storage_data.0))
 	}
 
 	fn get_storage_value_proof(
 		&self,
-		storage_prefix: &'static str,
-		storage_key_name: &'static str,
+		pallet: &'static str,
+		key: &'static str,
```

Comment:
```suggestion
		storage_item: &'static str,
```

Maybe this clarifies it a little better what the key is for? Not sure.

Reply:
```suggestion
		storage_name: &'static str,
```
Maybe we should do the naming like [here in subxt](https://github.com/paritytech/subxt/blob/92ace0629a63fadeebd428c5b1356ba60eeabfeb/metadata/src/lib.rs#L249)? So `pallet_name` and `storage_name`?